.github/workflows/scripts/e2e/manifest_gaudi_test.sh

```diff
@@ -205,6 +208,7 @@ fi
 
 case "$1" in
     init_docsum)
+        cp manifests/ChatQnA/qna_configmap_gaudi.yaml manifests/DocSum/gaudi/
```

Comment:
Hi @mkbhanda, because we want to reuse the template yaml files from ChatQnA which uses the configmap to inject environment variables into template yaml files.